src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -996,9 +996,7 @@ InsertMetadataForCitusLocalTable(Oid citusLocalTableId, uint64 shardId)
 	Assert(shardId != INVALID_SHARD_ID);
 
 	char distributionMethod = DISTRIBUTE_BY_NONE;
-	char replicationModel = ReplicationModel;
-
-	Assert(replicationModel != REPLICATION_MODEL_2PC);
+	char replicationModel = REPLICATION_MODEL_COORDINATOR;
```

Comment:
shouldn't this be STREAMING?

Reply:
This makes sense. If I understand it correctly, only deprecated `master_create_X_table` UDFs should be able to create tables with `REPLICATION_MODEL_COORDINATOR`

Reply:
We had a chat with @onurctirtir on his work on local tables, and he said that I can continue using coordinator replication. I think it makes more sense to use streaming replication instead.

@onurctirtir wrote a test that implies that all local tables should have 'c' replication model.

https://github.com/citusdata/citus/blob/d0ba122061245e85e9e63f9a3e66f8ce46fbedef/src/test/regress/sql/citus_local_tables.sql#L375-L380